docs/eventing/sinks/job-sink.md

```diff
@@ -0,0 +1,305 @@
+# JobSink, triggering long-running background jobs when events occurs
+
+Usually event processing combined with a Knative Service is expected to complete in a relative short
+period of time (minutes) as it requires the HTTP connection to stay open as otherwise the service is
+scaled down.
+
+Keeping long-running connections open increases the possibility of failing and so
+the processing needs to restart as the request is retried.
+
+This limitation is not ideal, `JobSink` is a resource you can use to create long-running
+asynchronous jobs and tasks.
+
+`JobSink` supports the full
+Kubernetes [batch/v1 Job resource and features](https://kubernetes.io/docs/concepts/workloads/controllers/job/)
+and Kubernetes Job queuing systems like [Kueue](https://kueue.sigs.k8s.io/).
```

Comment:
Maybe mention something along the lines of `except the .metadata.name field`? As far as I know, we set the name ourselves

Reply:
I don't think this is a detail we can add here at the introduction (and also is kinda intuitive if jobs are created dynamically a name can't be fixated)